### DISCUSSION TITLE
How to place the page content to

### DISCUSSION THREAD

**Discussion by junevm** [requarks/wiki#5134] [Help / Questions] [answered]
![Screenshot 2022-04-03 133709](https://user-images.githubusercontent.com/62926341/161418486-79b527ba-87be-48ef-bcfe-45233231c3ae.png)

**Comment by NGPixel** [comment, marked as answer]
Not possible atm. This is coming in v3.

**Comment by junevm** [comment]
ok by the way what is v3 release date

**Reply by 573077632** [reply]
2022H2
